tools/download_solc.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+source="https://github.com/FISCO-BCOS/solidity/releases/download"
+install_path="${HOME}/.fisco"
+version="0.4.25"
+OS="linux"
+crypto=
+
+LOG_WARN()
+{
+    local content=${1}
+    echo -e "\033[31m[WARN] ${content}\033[0m"
+}
+
+LOG_INFO()
+{
+    local content=${1}
+    echo -e "\033[32m[INFO] ${content}\033[0m"
+}
+
+help() {
+    echo $1
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/gh/FISCO-BCOS/go-sdk/pullRequest?prid=5307065)